### ISSUE TITLE
🐛 [BUG] - cannot reset or cancel a workout

### ISSUE THREAD

**Issue by jwallet** [madmustachecompany/Blast-Workout-App#46]
### Description

I can only stay in an active workout session or complete it.

A user might misclicked the Start Workout button and did not mean to start it right away. If the user leaves the workout by completing it and this workout was a pending challenge it will be removed from his list and deleting the workout form his logs/records does not resolve the issue. It's lost

### Reproduction steps

```bash
1. Generates workouts
1. Select a pending workout
1. Start it
1. Try to leave it with the back button. It does not stop the workout
1. Go back in it and try to leave it. The only option is Complete it. Click on it.
1. Your workout is completed and archived. Cannot be reset and kept in your pending workouts and at the right position.

Cloning it using save as a new routine is not what I'm looking for it.
```

### OS

Android

**Comment by noelielc** [comment]
You're right,  it's not possible to abandon a workout that's already started.
A workaround for the moment is to validate all the sets and then (on the "completed" page of the workout) tap on "add to pending" before deleting the completed workout (in the top right drop-down menu option).
